libs/newsletters-data-client/src/lib/schemas/newsletter-data-type.ts

```diff
@@ -132,6 +132,10 @@ export const newsletterDataSchema = z.object({
 	cancellationTimeStamp: z.number().optional(),
 
 	seriesTag: z.string().optional().describe('Series tag'),
+	seriesTagDescription: z
```

Comment:
I think there was also a request to include the a field for the name of the section the series tag should be under

Reply:
Yeah - adding a couple more things an another PR